modules/solid_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -70,7 +71,11 @@ CrackFrontDefinition::CrackFrontDefinition(const InputParameters & parameters) :
       mooseError("q_function_rings not supported with crack_front_points");
   }
   else if (isParamValid("boundary"))
+  {
     _geom_definition_method = CRACK_FRONT_NODES;
+    if (parameters.isParamSetByUser("closed_loop"))
+      mooseError("In CrackFrontDefinition, if 'boundary' is defined, 'closed_loop' should not be set by user!");
+  }
```

Comment:
So this parameter should _never_ be set by the user? The right way to do this would be to add a "private" parameter that way it's not displayed to the user in Peacock, Atom or any other file input helper.